uber/site_sections/registration.py

```diff
@@ -167,13 +178,13 @@ def undo_mpoint_cashout(self, session, id):
         return 'MPoint usage deleted'
 
     @ajax
```

Comment:
What the heck..? I fixed these bugs. I thought that code was included in this branch. How odd. Anyway, good catch!